client/src/components/Header.tsx

```diff
@@ -23,37 +23,52 @@ export default function Header() {
     dispatch(initializeWeb3());
   };
   return (
-    <header className="bg-light-primary text-light-primary dark:bg-dark-primary dark:bg-dark-primary-background">
-      {/* TODO: update with right logo */}
-      {/* <img
+    <header className="flex justify-between container mx-auto">
+      <div className="flex">
+        <img
+          className="py-4"
           alt="Gitcoin Logo"
           src={`${process.env.PUBLIC_URL}/assets/gitcoin-logo.svg`}
```

Comment:
@schultztimothy since we are using the hash history, we know the page itself is always at the root folder, maybe it's better to just use `./assets/...` wdyt?

Reply:
I like it!